packages/analytics-v1.1/src/core/analytics.js

```
@@ -1658,6 +1658,12 @@
 const endSession = instance.endSession.bind(instance);
 const setAuthToken = instance.setAuthToken.bind(instance);
 
+if ('__MODULE_TYPE__' === 'npm') {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)